recipe/meta.yaml

```diff
@@ -31,6 +33,11 @@ requirements:
 test:
   imports:
     - demes
+  commands:
+    - pip check
```

Comment:
Why are we doing this? Seems odd to invoke pip inside in a conda recipe. I would advise against this unless we're following some specific packaging methodology from conda-forge.

Reply:
I ran `grayskull pypi demes` and this was the output:
```yaml
{% set name = "demes" %}
{% set version = "0.2.0" %}


package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/demes-{{ version }}.tar.gz
  sha256: 2bc650cc0e60cebabc24bf00521a5833769f56b118f6ab44526e42171c961231

build:
  number: 0
  noarch: python
  entry_points:
    - demes = demes.__main__:cli
  script: {{ PYTHON }} -m pip install . -vv

requirements:
  host:
    - pip
    - python >=3.6
    - setuptools
    - setuptools-scm
  run:
    - attrs >=20.3.0
    - python >=3.6
    - ruamel.yaml >=0.15.78

test:
  imports:
    - demes
  commands:
    - pip check
    - demes --help
  requires:
    - pip

about:
  home: https://github.com/popsim-consortium/demes-python
  summary: tools for describing demographic models
  license: MIT
  license_file: LICENSE

extra:
  recipe-maintainers:
    - grahamgower
```

So I guess its normal to do this?

Reply:
Aha, nice! Sure, let's do it then.